src/test/perf_counters.cc

```diff
@@ -255,3 +257,369 @@ TEST(PerfCounters, read_avg) {
   t2.join();
   t1.join();
 }
+
+static PerfCounters* setup_test_perfcounter4(std::string name, CephContext *cct)
+{
+  PerfCountersBuilder bld(cct, name,
+	  TEST_PERFCOUNTERS2_ELEMENT_FIRST, TEST_PERFCOUNTERS2_ELEMENT_LAST);
+  bld.add_u64(TEST_PERFCOUNTERS2_ELEMENT_FOO, "foo");
+  bld.add_time(TEST_PERFCOUNTERS2_ELEMENT_BAR, "bar");
+
+  PerfCounters* counters = bld.create_perf_counters();
+  cct->get_perfcounters_collection()->add(counters);
+  return counters;
+}
+
+TEST(PerfCounters, TestLabeledCountersOnly) {
+  constexpr std::string_view EMPTY_DUMP_FORMAT_RAW = R"({}
```

Comment:
Hate to belabor this point but the special constant naming rule only extends to variables with static storage duration.  Automatic variables, even if `constexpr`, are named as usual in most cases since they don't need to stand out -- functions are supposed to be small and focused:
```suggestion
  constexpr std::string_view empty_dump_format_raw = R"({}
```